example/App.tsx

```diff
@@ -29,18 +35,14 @@ function App(): React.JSX.Element {
     setItem(data.map((datum: Data) =>
       datum.label
     ).join(", "));
+  const onThemeSelect = (datum: Data) =>
+    setTheme(datum.label as 'light' | 'dark' | 'system');
```

Comment:
This `as 'light' | 'dark' | 'system'` isn't really necessary since we already have the labels all lowercase, but I think it's nice to emphasize these are the only options there are.